frontend/src/apiMocks/handlers.ts

```diff
@@ -183,6 +184,7 @@ export function getHandlers(
     const id = (ownAddresses[ownAddresses.length - 1]?.id ?? -1) + 1;
     ownAddresses.push({
       address: body.address ?? `custom_alias_${id}`,
+      full_address: body.address ?? `custom_alias_${id}@mydomain.mozmail.com`,
```

Comment:
This doesn't add a domain if a domain name is passed in :)

```suggestion
      full_address: (body.address ?? `custom_alias_${id}`_ + "@mydomain.mozmail.com",
```

(Also don't forget line 120 :) )